File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/command/uielement/InputUi.java
package com.bigboxcode.designpattern.command.uielement;

public class InputUi implements UiCommand {
    @Override
    public void print() {
        System.out.println("Printing Input");
    }

    @Override
    public void remove() {
        System.out.println("Removing Input");
    }
}

File: design-patterns/php/command/uielement/InputUi.php
<?php
// InputUi.php

namespace BigBoxCode\DesignPattern\Command\UiElement;


class InputUi implements UiCommand {
    public function print(): void {
        echo "Printing Input\n";
    }

    public function remove(): void {
        echo "Removing Input\n";
    }
}